front-end/packages/robustar/src/views/ImageList.vue

```diff
@@ -432,16 +432,12 @@ export default {
     this.imagePerPage = this.imagePerPageOptions[1];
     this.initImageList();
     this.initClassifiedImageList();
-    this.initClassifiedImageList();
   },
   watch: {
     $route() {
       this.handleRouteChange();
       this.initImageList();
       this.initClassifiedImageList();
-
-      this.initClassifiedImageList();
-
```

Comment:
`initClassifiedImageList` function also seems to be defined multiple times. I think this is because of a bad merge. Could you remove the duplicate definition as well?